app/actions/package_delete.rb

```diff
@@ -23,5 +19,17 @@ def delete(packages)
 
       []
     end
+
+    private
+
+    def create_package_source_deletion_job(package)
```

Comment:
These ternaries are a little hard for me to read but I think I've worked it out:

If a package registry isn't configured, it returns a BlobstoreDelete job
If a package registry is configured *and* there are package bits, it returns a RegistryDelete job
If a package registry is configured but there are no package bits, it returns nil

Reply:
Yeah, that's right. One minor clarification though is that instead of "there are package bits" it is actually "this package is a bits package" (as opposed to a "docker package").

I'd really like the logic here to be:

* If `package.bits?`
  * If package registry configured: return `RegistryDelete`
  * Else return `BlobstoreDelete` job
* Else return nil

But the original behavior was to always return a `BlobstoreDelete` job and I was scared to change that behavior. See the convo with Connor above: https://github.com/cloudfoundry/cloud_controller_ng/pull/1909#discussion_r505910827